conf/multipart_id_numbering.conf

```diff
@@ -29,6 +28,216 @@ formats = {
+					description = _(Accession Prefix)
+				},
+				accession_year = {
+					type = YEAR,
+					editable = TRUE,
+					width = 4,
+					description = _(Accession Year),
+					force_derived_values_to_current_year = 0
+				},
+				year_number_separator = {
+					type = CONSTANT,
+					value = .,
+					width = 1,
+					description = _(Year Number Separator)
+				},
+				accession_number = {
+					type = NUMERIC,
+					editable = TRUE,
+					minimum_value = 1,
+					minimum_length = 1,
+					maximum_length = 5,
+					width = 5,
+					description = _(Accession Number)
+				},
+				accession_series_and_parts = {
+					type = FREE,
+					editable = TRUE,
+					minimum_length = 0,
+					maximum_length = 9,
+					width = 9,
+					description = _(Accession Series and Parts)
+				}
+			}
+		},
+		museumArtefact = {
+			# eg MA1985.85.89, MA1937.90.37a, MA2015.1, MA2015.7a-l
+			separator =,
+			elements = {
+				accession_prefix = {
+					type = CONSTANT,
+					value = MA,
+					width = 2,
+					description = _(Accession Prefix)
+				},
+				accession_year = {
+					type = YEAR,
+					editable = TRUE,
+					width = 4,
+					description = _(Accession Year),
+					force_derived_values_to_current_year = 0
+				},
+				year_number_separator = {
+					type = CONSTANT,
+					value = .,
+					width = 1,
+					description = _(Year Number Separator)
+				},
+				accession_number = {
+					type = NUMERIC,
+					editable = TRUE,
+					minimum_value = 1,
+					minimum_length = 1,
+					maximum_length = 5,
+					width = 5,
+					description = _(Accession Number)
+				},
+				accession_series_and_parts = {
+					type = FREE,
+					editable = TRUE,
+					minimum_length = 0,
+					maximum_length = 9,
+					width = 9,
+					description = _(Accession Series and Parts)
+				}
+			}
+		},
+		photograph = {
+			# eg P1999.1024.53, P1999.1035.54a, P2016.15, P2015.30a-b,P2015.147FP
+			separator =,
+			elements = {
+				accession_prefix = {
+					type = CONSTANT,
+					value = P,
+					width = 1,
+					description = _(Accession Prefix)
+				},
+				accession_year = {
+					type = YEAR,
+					editable = TRUE,
+					width = 4,
+					description = _(Accession Year),
+					force_derived_values_to_current_year = 0
+				},
+				year_number_separator = {
+					type = CONSTANT,
+					value = .,
+					width = 1,
+					description = _(Year Number Separator)
+				},
+				accession_number = {
+					type = NUMERIC,
+					editable = TRUE,
+					minimum_value = 1,
+					minimum_length = 1,
+					maximum_length = 5,
+					width = 5,
+					description = _(Accession Number)
+				},
+				accession_series_and_parts = {
+					type = FREE,
+					editable = TRUE,
+					minimum_length = 0,
+					maximum_length = 9,
+					width = 9,
+					description = _(Accession Series and Parts)
+				}
+			}
+		},
+		publicMemorial = {
+			# eg MA1937.90.37a, MA2015.7a-l, MA2015.40ab, MA2013.34a, MA2015.2
+			separator =,
+			elements = {
+				accession_prefix = {
+					type = CONSTANT,
+					value = MA,
```

Comment:
Copy pasta?  Or do memorials and artefacts have the same prefix?  (Also on line 201).